stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -209,6 +209,9 @@ end
 @test norm([2.4e-322, 4.4e-323], 3) ≈ 2.4e-322
 @test_throws ArgumentError opnorm(Matrix{Float64}(undef,5,5),5)
 
+# operator norm for zero-dimensional domain is zero (see #40370)
+@test opnorm(fill(1,1,1)[:,2:1]) == 0.0
```

Comment:
Probably should test both `m == 0` and `n == 0`.

Also, if you are special-casing the `m == 1 || n == 1` case, those should both be tested.